app/views/shared/_menu.html.erb

```diff
@@ -1,9 +1,15 @@
 <div style="text-align: right; padding-right: 3rem">
 
   <nav>
-  <a href="/users/new">Sign up</a> |
-  <a href="/login">Log in</a> |
-  <a href="/spaces/new">Add your space</a> |
-  <a href="/spaces/show">View spaces</a>
+  <% links = {
+  'Sign up' => '/users/new',
```

Comment:
I like it - this hash looks to me like it might make a good constant passed to the view though - what do you an others think?